scripts/travis/prepare-integration.sh

```diff
@@ -16,6 +16,9 @@ st2 --version
 # Clean up old screen log files
 rm -f logs/screen-*.log
 
+# ::group::/::endgroup:: is helpful github actions syntax to fold this section.
+echo ::group::launchdev.sh start -x
```

Comment:
As per comment above since that script is now specific to gha, we should probably rename the directory and update the affected files :)

Reply:
It's not very difficult to make this work across multiple CI providers (like I did in https://github.com/ansible-community/molecule/pull/2976 ), but I didn't see the point since this will only be running in GHA for now.

As I don't want to disable Travis until after this PR is merged, I minimized my changes to the travis scripts. Once we disable travis, though, there's a lot of code on the chopping block.